source/math/Quaternion.ooc

```diff
@@ -33,7 +33,7 @@ Quaternion: cover {
 	// z = this y
 	// w = this z
 
-	inverse ::= This new(this w, -this x, -this y, -this z)
+	inverse ::= This new(this w / (this norm * this norm), -this x / (this norm * this norm), -this y / (this norm * this norm), - this z / (this norm * this norm))
```

Comment:
I don't know whether `inverse` or `conjugate` is truly what we want in any given situation, but for now, let's assume that everything relies on the current implementation of `inverse` being what it is and will break if we change its implementation. It's entirely possible that the correct implementation is what we want in all of those cases, but let's not guess.